libs/@blockprotocol/service/src/types.ts

```diff
@@ -241,7 +241,10 @@ export type MapboxRetrieveStaticMapResponseData = {
   data: ArrayBuffer;
 };
 
-export type ServiceMessageError = "FORBIDDEN";
+export type ServiceMessageError =
```

Comment:
I think we should have "NOT_IMPLEMENTED" within this

Reply:
I have added this and other error codes in https://github.com/blockprotocol/blockprotocol/pull/1049/commits/fef025d0a3c293b5bb2af5fefd6b0d074b77c977